host/src/FunctionsCustomHost/FunctionsCustomHost.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+    <PublishAot>true</PublishAot>
+    <OptimizationPreference>Speed</OptimizationPreference>
+    <IlcExportUnmanagedEntrypoints>true</IlcExportUnmanagedEntrypoints>
```

Comment:
@mattchenderson 
What should be the executable name be? Should it be "func" or Can we use another name? I see we have the `localEntryPoint` property value in the feed item, which I am assuming is what VS uses to launch the exe. 

@aishwaryabh  You can set the `AssemblyName` element to change the executable name produced.

Reply:
Spoke to @mattchenderson  and @fabiocav . The executable name should be `func`.

Reply:
changed to func. let me know if it looks ok